tools/cooja/apps/mspsim/build.gradle

```diff
@@ -0,0 +1,61 @@
+apply plugin: "java"
+
+sourceSets {
+    main {
+        java {
+            srcDir 'src'
+        }
+    }
+}
+
+task buildMspsimJar(type: GradleBuild) {
+    buildFile = 'importedmspsimBuild.gradle'
+    tasks = ['jar']
+}
+task buildCoffeeJar(type: GradleBuild) {
+    buildFile = '../../../coffee-manager/build.gradle'
```

Comment:
It might be nice to replace `../../../` with some variable like `contiki_tools_path`, but that's eye-candy